parking_permits/views.py

```diff
@@ -42,9 +42,15 @@ class TalpaResolveAvailability(APIView):
         tags=["ResolveAvailability"],
     )
     def post(self, request, format=None):
+        logger.info(
+            f"Data received for resolve availability = {json.dumps(request.data)}"
```

Comment:
I don't know if this is a concern, but if I'm not mistaken, the request data contains information about customer, which is not tracked and deletable via the GDPR API.

Reply:
It contains only the userId of the customer. So I guess that should not be a problem for GDPR as if you delete the user the ID in the logs doesn't make sense